ports/mimxrt10xx/common-hal/busio/I2C.c

```diff
@@ -187,9 +179,6 @@ bool common_hal_busio_i2c_probe(busio_i2c_obj_t *self, uint8_t addr) {
 }
 
 bool common_hal_busio_i2c_try_lock(busio_i2c_obj_t *self) {
-    if (common_hal_busio_i2c_deinited(self)) {
```

Comment:
try_lock could be used from displayio. I think it was recently changed to relying on this returning failure if the bus was deinited. so, I don't think this can be removed.

Reply:
I'm not sure why this disappeared. The others were fine. Thanks for noticing this.